bedrock/settings/base.py

```diff
@@ -485,7 +485,11 @@ def language_url_map_with_fallbacks():
     "xbl",  # in mozorg urls
     "revision.txt",  # from root_files
     "locales",  # in mozorg urls
+    "404",
+    "500",
```

Comment:
**Question**: I can view http://localhost:8000/csrf_403/ OK here, but http://localhost:8000/404/ redirects to the (localized) 404 template preview URL at http://localhost:8000/en-US/404/. Is that expected?

Same thing happens for `/500/`

Reply:
Yeah - i made a decision to make the csrf_403 view deliberately unlocalised because it'll only get seen by MoCo staff, who all speak English